assets/stylesheets/components/_button.scss

```diff
@@ -3,6 +3,7 @@
 	color: $white;
 	padding:8px 32px 7px;
 	display: inline-block;
+	text-transform: uppercase;
```

Comment:
This looks great for the "FREE QUOTE" button, but almost seems too aggressive on the "See what we offer" callout - given the services and products we're selling. Consistency might be more important than that though.